lib/time.py

```diff
@@ -23,3 +55,37 @@ def sleep(secs):
 
 def time():
   return float(Now().UnixNano()) / Second
+
+
+_strftime_map = {
+    'Y': '2006', 'y': '06', 'b': 'Jan', 'B': 'January', 'm': '01', 'd': '02',
+    'a': 'Mon', 'A': 'Monday',
+    'H': '15', 'I': '03', 'M': '04', 'S': '05', 'L': '.000',
+    'p': 'PM',
+    'Z':  'MST', #'z': '-0700',
+    '%': '%',
+    # TODO: Implement c, x, X, j, U, w, W
+}
+
+def strftime(layout, tt=None): #pylint: disable=missing-docstring
```

Comment:
I will change that, I didn't use it because format seems to be python's builtins function.